src/xpk/core/docker_image.py

```diff
@@ -26,6 +26,7 @@
 DEFAULT_DOCKER_IMAGE = 'python:3.10'
 DEFAULT_SCRIPT_DIR = os.getcwd()
 PLATFORM = 'linux/amd64'
+CLOUD_PREFIXES = ['gcr.io', 'docker.pkg.dev', 'us-docker.pkg.dev']
```

Comment:
Technically you can have zone specific prefixes too like `us-west1-docker.pkg.dev`, we can just make it so that it checks the regex like `.*-docker.pkg.dev` as well